booking/src/main/resources/standards/booking/messages/booking-api-2.0.0-regular.json

```diff
@@ -53,15 +53,35 @@
   "shipmentLocations": [
     {
       "location": {
-        "locationType": "UNLO",
-        "UNLocationCode": "POL_UNLOCATION_CODE_PLACEHOLDER"
+        "UNLocationCode": "POL_UNLOCATION_CODE_PLACEHOLDER",
+        "address": {
+          "name": "Port of Rotterdam",
+          "street": "Europoort Rotterdam",
+          "postCode": "3198 LK",
+          "city": "Rotterdam",
+          "countryCode": "NL"
+        },
+        "geoCoordinate": {
+          "latitude": "53.551086",
+          "longitude": "9.993682"
+        }
```

Comment:
If we have this data, it should be a placeholder otherwise it will not match the UNLocationCode.

I recommend we do not have it for exactly that reason.

Reply:
Note: It is fine for the "all attributes" message but not for the base API message used in all other tests (Given it was called `-regular` I assumed it was for the regular tests)

Reply:
I think it doesnt make sense to all addresses. I will remove them